frontend/src/components/Playback/PlaybackInfoCard.vue

```diff
@@ -1,427 +0,0 @@
-<template>
```

Comment:
This is not unused, it's pending the reimplementation of the video player (#1878)

Reply:
Unused in the sense that the current branch didn't have any references to it :)
ThePR you referenced doesn't change it (yet) and a lot of the component logic is pretty dependent on the old shaka player types. Would you prefer I leave it untouched or should I clean it out so that it's not relying on the shaka player types?

Reply:
That should be answered by @ThibaultNocchi :)

Reply:
Going to remove this from the PR for now and follow up on Matrix